packages/block-library/src/navigation/edit.js

```diff
@@ -33,64 +34,28 @@ import BlockColorsStyleSelector from './block-colors-selector';
 import * as navIcons from './icons';
 import NavigationPlaceholder from './placeholder';
 
+const ref = createRef();
```

Comment:
Why do we want to use `createRef`? I think it's dangerous since now every instance will point to the same global ref defined here. This prevents `<Navigation>` from rendering more than once I believe 🤔.